backend/editor/entries.py

```diff
@@ -44,6 +54,104 @@ def create_node(self, label, entry, main_language_code):
         result = get_current_transaction().run(" ".join(query), params)
         return result
 
+    def parse_taxonomy(self, filename):
+        """
+        Helper function to call the Open Food Facts Python Taxonomy Parser
+        """
+        # Close current transaction to use the session variable in parser
+        get_current_transaction().commit()
+
+        # Create parser object and pass current session to it
+        parser_object = parser.Parser(get_current_session())
+        try:
+            # Parse taxonomy with given file name and branch name
+            parser_object(filename, self.branch_name, self.taxonomy_name)
+            return True
+        except:
+            return TaxonomyParsingError()
+
+    def import_from_github(self, description):
+        """
+        Helper function to import a taxonomy from GitHub
+        """
+        base_url = "https://raw.githubusercontent.com/openfoodfacts/openfoodfacts-server/main/taxonomies/"
+        filename = self.taxonomy_name + '.txt'
+        base_url += filename
+        try:
+            # Downloads and creates taxonomy file in current working directory
+            urllib.request.urlretrieve(base_url, filename)
+            status = self.parse_taxonomy(filename) # Parse the taxonomy
+            self.create_project(description) # Creates a "project node" in neo4j
+            os.remove(filename) # Removes taxonomy file after parsing
+            return status
+        except:
+            return TaxnonomyImportError()
+    
+    def export_taxonomy(self):
+        """
+        Helper function to export a taxonomy for download
+        """
+        # Close current transaction to use the session variable in unparser
+        get_current_transaction().commit()
+
+        # Create unparser object and pass current session to it
+        unparser_object = unparser.WriteTaxonomy(get_current_session())
+        filename = self.taxonomy_name + '.txt'
+        try:
+            # Parse taxonomy with given file name and branch name
+            unparser_object(filename, self.branch_name, self.taxonomy_name)
+            return filename
+        except:
+            return TaxonomyUnparsingError()
+
+    
+    def check_if_project_exists(self):

```

Comment:
picky/personal preference: 

as this return True or False, it's better named `project_exists` 
so that you can read  `if (taxonomy.project_exist())`
When you use check_.... I would expect it to raise instead if it's not ok.

same below: `is_branch_unique`

Reply:
Yep @alexgarel, that's actually a good choice. I'll do the same from now on :)